xml/admin_saltcluster.xml

```diff
@@ -1141,14 +1141,29 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
    </step>
    <step>
     <para>
-     If the &adm; is a &smaster; at the same time, stop
-     <systemitem class="daemon">salt-master</systemitem>
-     <systemitem class="daemon">salt-minion</systemitem> services on the old
-     &adm;:
+     If the old &smaster; is also the only administration node in the cluster
+     <!-- see <xref linkend="move-admin-node" for more information. -->, then
+     manually move <filename>etc/ceph/ceph.client.admin.keyring</filename> and
+     <filename>/etc/ceph/ceph.conf</filename> to the new &smaster;.
```

Comment:
Missing a slash here should be `<filename>/etc/ceph/`

Reply:
Maybe I'm blind but that is what's there? `/etc/ceph/ceph.conf`

Reply:
> Maybe I'm blind

Not in the 1st one:
<filename>etc/ceph/ceph.client.admin.keyring</filename>